schema/compose-spec.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://json-schema.org/draft/2019-09/schema#",
-  "id": "compose_spec.json",
+  "$id": "https://raw.githubusercontent.com/compose-spec/compose-spec/HEAD/schema/compose-spec.json",
```

Comment:
:+1
fixed earlier by https://github.com/compose-spec/compose-go/commit/25fe17966316466351cf15d5c09c888e28a760e6 but unfortunately mixed other changes we had to revert